e2e/provision/gce_init.sh

```diff
@@ -30,6 +30,11 @@ BRANCH=$(get_metadata nephio-test-infra-branch "main")
 
 echo "$DEBUG, $DEPLOYMENT_TYPE, $RUN_E2E, $REPO, $BRANCH"
 
+# pre-configure the "docker" group and add the "ubuntu" user to it so that the docker
+# and kpt fn apply commands will work without sudo
+addgroup docker
```

Comment:
@liamfallon looks like @johnbelamaric already has a PR to achieve this -> https://github.com/nephio-project/test-infra/pull/70/files#diff-12c142480c5cefec671f639ba984f93a1c3baac9a52e775532b8a2cd69ca4da2R56